src/nspawn/nspawn-mount.c

```diff
@@ -275,20 +288,22 @@ int mount_all(const char *dest,
                 unsigned long flags;
                 bool fatal;
                 bool userns;
+                bool netns;
```

Comment:
I don't like the naming `userns` and `netns`... The semantics of those two booleans are not the same. Any suggestions?